service/javadsl/kafka/client/src/main/java/com/lightbend/lagom/javadsl/broker/kafka/KafkaTopicFactory.java

```diff
@@ -26,24 +27,31 @@
     private final ActorSystem system;
     private final Materializer materializer;
     private final ExecutionContext executionContext;
-    private final KafkaConfig config;
+    private final KafkaConfig kafkaConfig;
     private final ServiceLocator serviceLocator;
 
+    /**
+     * @deprecated As of release 1.5.0. Use {@link #KafkaTopicFactory(ServiceInfo, ActorSystem, Materializer, ExecutionContext, ServiceLocator, Config)} instead.
```

Comment:
Oh, I approved and merged this one. 

We need to reconsider this. Is the next release 1.6.0 or 2.0.0? In any case, not 1.5.0.

I send a PR on this and we can then decide?

Reply:
Here is the PR to fix/discuss it. 
https://github.com/lagom/lagom/pull/1882

Relates with discussion at: https://github.com/lagom/lagom/pull/1880#discussion_r278737897